core/src/main/java/org/elasticsearch/action/search/SearchTransportService.java

```diff
@@ -390,7 +389,18 @@ public void messageReceived(ShardFetchSearchRequest request, TransportChannel ch
         TransportActionProxy.registerProxyAction(transportService, FETCH_ID_ACTION_NAME, FetchSearchResult::new);
     }
 
-    Transport.Connection getConnection(DiscoveryNode node) {
-        return transportService.getConnection(node);
+    /**
+     * Returns a connection to the given node on the provided cluster. If the cluster is <code>null</code> the node will be resolved
+     * against the local cluster.
+     * @param clusterAlias the cluster alias the node should be resolve against
+     * @param node the node to resolve
+     * @return a connection to the given node on the give cluster alias.
```

Comment:
a connection to the given node belonging to the cluster with the provided alias  ?